src/components/custom/dashboard/client/pages/diet-tracker/AddSnackModal.jsx

```
@@ -0,0 +1,784 @@
+                placeholder="e.g., Apple, Chocolate Bar, Protein Shake"
+                error={errors.name}
+                required
+                className="w-full"
+              />
+
+              {/* Snack Type Selection */}
+              <div className="space-y-2">
+                <label className="block text-sm font-medium text-white">
+                  Type
+                </label>
+                <div className="grid grid-cols-2 sm:grid-cols-3 gap-3">
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)